index.js

```diff
@@ -457,7 +457,11 @@ const VIEWPORT_FUDGE_FACTOR = -3; // px
 const MIN_VIEW_TIME_ROTATION_DURATION = 45; // seconds
 const MAX_ROTATIONS = 3;
 
+// Enable ad rotation on hash change (intra-site nav)
+const HASHCHANGE_ROTATION_ENABLE = true;
```

Comment:
Would be nice if we could set this in the decision API response, so we could enable it dynamically, even per publisher 🤔

Reply:
I do like that idea. I'll probably skip it here but I'm going to be working on saving more ad rotation details in the server and that's a good time to work on those features.